README.md

```diff
@@ -50,7 +50,7 @@ Open the terminal with Ctrl+Alt+T and type `shell`. If this command returns `ERR
 Then download and run the installation script below:
 
 ```bash
-curl -Ls git.io/vddgY | bash
+curl -O -Ls git.io/vddgY && bash ./vddgY
```

Comment:
Why the change?  What you suggest is a bit more intrusive since the script is actually downloaded and left there.

Reply:
It's required to do it this way for the script to be able to take user input.

Reply:
This will break installation on read-only filesystems like `/`. Just redirect input back to TTY when calling `crew` or running something that need input:
```shell
crew ... < /dev/tty
```

Reply:
Not sure why people would be running the install command on `/`, but I would like to be able to do it the original way.

Only one bit requires user input, and thats the question about clearing out `CREW_PREFIX`:
```bash
    select continue in "Yes" "No"; do
      if [[ "${continue}" == "Yes" ]]; then
        sudo find "${CREW_PREFIX}" -mindepth 1 -delete
        break 2
      else
        exit 1
      fi
    done
```

What exactly should I do here? I tried a few permutations of `< /dev/tty`, but no luck.

Reply:
The initial directory of `crosh> shell` is in `/`...

Maybe try this (don't know about `select ... do` before😅):
```shell
{
  select ...
} < /dev/tty
```

Reply:
(See my other comment, `exec bash --init-file <(curl -Ls git.io/vddgY)` allows us to take user input)